src/yival/enhancers/utils.py

```diff
@@ -28,22 +28,33 @@ def scratch_variations_from_str(
     target_str:
         This is the generated new output
         var1: hello world
+        hello world
         var2: bye
     
     variations: ["var1", "var2"]
 
     function result:
     {
-        "var1": "hello world"
+        "var1": "hello world\nhello world"
         "var2": "bye"
     }
     """
     result = {}
     lines = target_str.split('\n')
+    var_detect = False
+    var_now = ""
     for line in lines:
+        var_line = False
         for var in variations:
             if line.startswith(var + '='):
+                var_detect = True
                 result[var] = line[len(var) + 1:].strip().strip("'").strip('"')
```

Comment:
- Instead of hardcoding len(var)+1, I think searching the frist '=' is more safe. How do you think? I'm not sure here
- Simplify .strip("'").strip('"') to .strip("'\"")
`# Split the line at the first occurrence of '=' and take the second part`
`result[var] = line.split('=', 1)[1].strip().strip("'\"")`

Reply:
yeah , that's more clear, definitely